doc/requirements.txt

```diff
@@ -31,4 +31,4 @@ sphinxcontrib-qthelp==1.0.3
 sphinxcontrib-serializinghtml==1.1.5
 urllib3>=2.2.2
 zipp==3.19.1
-setuptools==75.1.0
+setuptools==78.1.1
```

Comment:
It looks like this PR (and some others) are already in master:


![image](https://github.com/user-attachments/assets/b37617c5-8a77-439b-a8f5-898e5e50eddc)


why is it added once again here ?

I can see the commit in the "commits" section but not in the "files changed".

This also applies to multiple other commits that edit the README file

Reply:
because internal branches 15.xxx are not aligned with uxl. you didnt have these questions to 15.1;)